backends/graphics/opengl/opengl-graphics.cpp

```diff
@@ -796,8 +796,8 @@ void OpenGLGraphicsManager::displayMessageOnOSD(const char *msg) {
 #ifdef USE_OSD
 void OpenGLGraphicsManager::osdMessageUpdateSurface() {
 	// Split up the lines.
-	Common::Array<Common::String> osdLines;
-	Common::StringTokenizer tokenizer(_osdMessageNextData, "\n");
+	Common::Array<Common::U32String> osdLines;
+	Common::StringTokenizer tokenizer(Common::convertFromU32String(_osdMessageNextData), "\n");
```

Comment:
Will you lose the U32 data here during the converting? Why?

Reply:
A U32Tokenizer has been implemented.